en/docs/learn/exporting-user-info-to-sendgrid.md

```diff
@@ -0,0 +1,29 @@
+#Exporting WSO2 Identity Server User Information to Sendgrid
+
+SendGrid is a cloud-based service that assists businesses with email delivery. 
+This topic provides instructions on exporting user data from WSO2 Identity Server (WSO2 IS) to Sendgrid Email Delivery
+platform.
+
+Let's get started!
+
+!!! tip "Before you begin!"
+    Create **users.csv** file by following the instructions in 
+    [Exporting User Data From WSO2 Identity Server](../learn/exporting-user-data-from-wso2-is.md).
+    
+##Try it Out
+
+1. Login to [Sendgrid](https://sendgrid.com/).
+2. Click **Contacts** under **Marketing**.
+   
+   ![contacts-in-sendgrid](../assets/img/tutorials/contacts-in-sendgrid.png)
+3. Click **Upload CSV** under **Add Contacts**.
+   
+   ![data-import-in-sendgrid](../assets/img/tutorials/data-import-in-sendgrid.png)   
+4. Select **Add Contacts and include in existing list**(ie. This may change according to your requirement) 
+   radio button and select the CSV file with user data.
+
+   ![import-from-spread-sheet-sendgrid](../assets/img/tutorials/import-from-spread-sheet-sendgrid.png)
+5. Map **CSV data fields** with **Sendgrid data fields**. **Skip** unwanted data fields.
+6. Click **Next: Add Contacts**.
```

Comment:
**Next > Add Contacts** 

Please check if this is how it should be.

Reply:
**Next: Add Contacts** is the correct one.